apps/s_client.c

```diff
@@ -1311,14 +1313,32 @@ int s_client_main(int argc, char **argv)
         }
     }
     argc = opt_num_rest();
-    if (argc != 0)
+    if (argc == 1) {
+        // If there's a positional argument, it's the equivalent of OP_CONNECT.
+#ifdef AF_UNIX
+        if (socket_family == AF_UNIX) {
```

Comment:
Ok, cool. I can open an issue to track this but I'm aware that OpenSSL has an internal bug tracker: would you rather I opened it here or there?